File: app/src/main/java/com/nusantarian/autoscad/Activity/EditProfileActivity.java
package com.nusantarian.autoscad.Activity;

import android.content.Intent;
import android.text.TextUtils;
import android.view.Menu;
import android.view.MenuItem;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;
import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import androidx.appcompat.widget.Toolbar;
import com.google.firebase.auth.FirebaseAuth;
import com.google.firebase.database.DatabaseReference;
import com.google.firebase.storage.StorageReference;
import com.nusantarian.autoscad.R;

import java.util.Objects;

public class EditProfileActivity extends AppCompatActivity implements View.OnClickListener {

    private FirebaseAuth mAuth;
    private DatabaseReference mDatabase;
    private StorageReference mStorage;
    private String nama, phone, email, uid;
    private EditText et_nama, et_phone, et_email;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_edit_profile);

        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        Objects.requireNonNull(getSupportActionBar()).setTitle(R.string.pengaturan_profil);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        mAuth = FirebaseAuth.getInstance();

        et_nama = findViewById(R.id.et_nama);
        et_email = findViewById(R.id.et_email);
        et_phone = findViewById(R.id.et_phone);
        findViewById(R.id.btn_change).setOnClickListener(this);
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.editprofile_menu, menu);
        return super.onCreateOptionsMenu(menu);
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        if (item.getItemId() == R.id.nav_changepass){
            startActivity(new Intent(EditProfileActivity.this, ChangePassActivity.class));
            overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
        }
        return super.onOptionsItemSelected(item);
    }

    @Override
    public void onClick(View view) {
        if (view.getId() == R.id.btn_change){
            nama = et_nama.getText().toString();
            email = et_email.getText().toString();
            phone = et_phone.getText().toString();


        }
    }
}

File: app/src/main/java/com/nusantarian/autoscad/activity/EditProfileActivity.kt
package com.nusantarian.autoscad.activity

import android.content.Intent
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.view.View
import android.widget.EditText
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import androidx.test.runner.AndroidJUnit4
import com.google.firebase.auth.FirebaseAuth
import com.google.firebase.database.DatabaseReference
import com.google.firebase.storage.StorageReference
import com.nusantarian.autoscad.R
import java.util.*

class EditProfileActivity : AppCompatActivity(), View.OnClickListener {
    private var mAuth: FirebaseAuth? = null
    private val mDatabase: DatabaseReference? = null
    private val mStorage: StorageReference? = null
    private var nama: String? = null
    private var phone: String? = null
    private var email: String? = null
    private val uid: String? = null
    private var et_nama: EditText? = null
    private var et_phone: EditText? = null
    private var et_email: EditText? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_edit_profile)
        val toolbar = findViewById<Toolbar?>(R.id.toolbar)
        setSupportActionBar(toolbar)
        Objects.requireNonNull(supportActionBar).setTitle(R.string.pengaturan_profil)
        supportActionBar.setDisplayHomeAsUpEnabled(true)
        mAuth = FirebaseAuth.getInstance()
        et_nama = findViewById(R.id.et_nama)
        et_email = findViewById(R.id.et_email)
        et_phone = findViewById(R.id.et_phone)
        findViewById<View?>(R.id.btn_change).setOnClickListener(this)
    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.editprofile_menu, menu)
        return super.onCreateOptionsMenu(menu)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == R.id.nav_changepass) {
            startActivity(Intent(this@EditProfileActivity, ChangePassActivity::class.java))
            overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
        }
        return super.onOptionsItemSelected(item)
    }

    override fun onClick(view: View?) {
        if (view.getId() == R.id.btn_change) {
            nama = et_nama.getText().toString()
            email = et_email.getText().toString()
            phone = et_phone.getText().toString()
        }
    }
}